source/beamline/CLS/CLSSSRLMonochromatorMaskState.cpp

```diff
@@ -1,8 +1,9 @@
-#include "BioXASSSRLMonochromatorMaskState.h"
+#include "CLSSSRLMonochromatorMaskState.h"
+
 #include "actions3/AMActionSupport.h"
 #include "actions3/AMListAction3.h"
 
```

Comment:
Hey @XuetaoLiu-CLS, why was an SSRL mono class made into a CLS class here? Is it going to be used somewhere else?